parachain/pallets/ethereum-beacon-client/fuzz/Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "snowbridge-ethereum-beacon-client-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = { version = "0.4", features = [ "arbitrary-derive" ]}
+arbitrary = { version = "1", optional = true, features = ["derive"] }
+snowbridge-ethereum-beacon-client = {path = "..", features = [ "fuzzing" ]}
+snowbridge-beacon-primitives= {path = "../../../primitives/beacon"}
+hex-literal = { version = "0.4.1" }
```

(nit) We can just set the version, same as the manifest in the pallet:
```suggestion
hex-literal = "0.4.1"
```